### DISCUSSION TITLE
Unban Watchdog from bossing areas, ban delayed notifications specifically.

### DISCUSSION THREAD

**Discussion by billy-bg** [adamk33n3r/runelite-watchdog#165] [Ideas]
The main feature of watchdog that was considered OP while bossing was always linked to the delayed notifications feature (such as being able to tick eat Sotetseg easily).
Banning the watchdog plugin completely makes some bossing scenarios very hard for people who hide chat/use the fixed mode hide chat plugin. It would make a lot more sense to ban the delayed notifications feature in bossing areas ONLY.
The main feature of watchdog itself does not confer any additional advantages over having the chatbox itself open. There are no additional advantages gained from a 1:1 non-delayed watchdog notification feature while bossing.

**Comment by adamk33n3r** [comment]
This was proposed during that time but unfortunately the powers that be did not approve. 
![image](https://github.com/user-attachments/assets/88d3325d-81ce-4b77-8022-d71172527986)

**Reply by billy-bg** [reply]
unfortunate, hopefully they can change their mind in the future. Thanks for your help.
